### PR TITLE
Feature/restyle homepage

### PR DESCRIPTION
**Pull request by arealclimber**
### DEVELOP

- [x] ReserveRatio 
- [x] NavBarMobile 
- [x] NavBar 
- [x] HeroDescription
- [x] Notification
-----
更改 modal 的 z-index

### CHECK LIST
 
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  2
    - src/components/audit_report/audit_report.tsx
    - src/components/reserve_card/reserve_card.tsx
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/qrcode_modal/qrcode_modal.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const QrcodeModal = ({
 
   const isDisaplayedQrcodeModal = qrcodeModalVisible ? (
     <>
-      <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">
+      <div className="fixed inset-0 z-80 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">
         <div className="relative my-6 mx-auto w-auto max-w-xl">
           {/*content & panel*/}
```
Comment (arealclimber) [inline src/components/qrcode_modal/qrcode_modal.tsx:24]: - 刪掉 comment

File: src/components/successful_modal/successful_modal.tsx [hunk 2]
```diff
@@ -71,9 +71,6 @@ const SuccessfulModal = ({
 
   const isDisplayedModal = modalVisible ? (
     <>
-      {/*  <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">*/}
-      {/*  overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none */}
-      {/* position: relative; top: 50%; left: 50%; transform: translate(-50%, -50%) */}
       <div className="fixed inset-0 z-70 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">
         {/* The position of the modal */}
```
Comment (arealclimber) [inline src/components/successful_modal/successful_modal.tsx:75]: - 刪掉 comment

File: src/components/ticker_selector_box/ticker_selector_box.tsx [hunk 3]
```diff
@@ -249,7 +249,7 @@ const TickerSelectorBox = ({
 
   const isDisplayedTickerSelectorBox = tickerSelectorBoxVisible ? (
     <>
-      <div className="fixed inset-0 z-50 hidden items-center justify-center overflow-x-auto overflow-y-auto outline-none backdrop-blur-sm focus:outline-none lg:flex">
+      <div className="fixed inset-0 z-80 hidden items-center justify-center overflow-x-auto overflow-y-auto outline-none backdrop-blur-sm focus:outline-none lg:flex">
         {/* The position of the box */}
```
Comment (arealclimber) [inline src/components/ticker_selector_box/ticker_selector_box.tsx:253]: - 刪掉 comment

File: src/components/wallet_panel/wallet_panel.tsx [hunk 4]
```diff
@@ -115,7 +115,7 @@ export default function WalletPanel({
 
   const walletPanel = (
     <>
-      <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">
+      <div className="fixed inset-0 z-80 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none backdrop-blur-sm focus:outline-none">
         <div className="relative my-6 mx-20px w-auto max-w-xl md:mx-auto">
           {/*content & panel*/}
```
Comment (arealclimber) [inline src/components/wallet_panel/wallet_panel.tsx:120]: - 刪掉 comment

File: src/lib/hooks/use_outer_click.ts [hunk 5]
```diff
@@ -1,12 +1,14 @@
 import React, {useState, useEffect, useRef, forwardRef} from 'react';
+import useStateRef from 'react-usestateref';
 
 // interface UseOuterClickProps { // initialVisibleState: boolean; // componentVisible: boolean; //
```
Comment (arealclimber) [inline src/lib/hooks/use_outer_click.ts:4]: - 刪掉 comment

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**